test/functional/cmdLineTests/gcRegressionTests/gcRegressionTests_excludes.xml

```diff
@@ -43,14 +43,6 @@
 <exclude id="Excessive GC throws OOM" platform="Mode301" shouldFix="true"><reason>Metronome and Staccato do not use excessive GC</reason></exclude>
 <exclude id="Excessive GC appears in verbose log" platform="Mode301" shouldFix="true"><reason>Metronome and Staccato do not use excessive GC</reason></exclude>
 
-<!-- Metronome does not cause multiple verboseGC logging files -->
-<!--
-<exclude id="GC rotating verbose log file name contains %s" platform="Mode301" shouldFix="false"><reason>Metronome does not cause multiple verboseGC logging files</reason></exclude>
-<exclude id="GC rotating verbose log file name contains %s and other random symbols" platform="Mode301" shouldFix="false"><reason>Metronome does not cause multiple verboseGC logging files</reason></exclude>
-<exclude id="GC rotating verbose log file name contains %s %c %i" platform="Mode301" shouldFix="false"><reason>Metronome does not cause multiple verboseGC logging files</reason></exclude>
-<exclude id="GC rotating verbose log file name contains %s %c %i and other random symbols" platform="Mode301" shouldFix="false"><reason>Metronome does not cause multiple verboseGC logging files</reason></exclude>
--->
-
 <!-- The Windows system also does not support the * character in file or directory names.  -->
 <!--
 <exclude id="GC rotating verbose log file name contains %s and other random symbols" platform="win_x86.*" shouldFix="false"><reason>Windows does not support * symbol</reason></include>
```

Comment:
We need to exclude the test on wins. The same for line 57.

Reply:
Created gcRotatingVerboseLogTests_excludes.xml for wins exclude, thanks.